PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClient.kt

```diff
@@ -247,16 +252,25 @@ internal class PayPalInternalClient(
 
     private fun getClientMetadataId(
         context: Context,
+        correlationId: String?,
         callback: (String?) -> Unit
     ) {
         braintreeClient.getConfiguration { configuration, error ->
             if (error != null) {
                 callback(error("No Client Metadata Id"))
             } else {
-                val clientMetadataId = dataCollector.getClientMetadataId(
-                    context,
-                    configuration,
+                val request = DataCollectorRequest(false, correlationId)
```

Comment:
We can address whether the param should be true/false in a future PR, but let's still make sure the implementation is correct in this PR aside from that parameter. This `request` object isn't currently being used - we should pass it to `dataCollector.collectDeviceData` rather than calling `dataCollector.getClientMetadataId` directly